INSTALL

```diff
@@ -561,12 +561,16 @@
                    CPPDEFINES      List of CPP macro definitions, separated
                                    by a platform specific character (':' or
                                    space for Unix, ';' for Windows, ',' for
-                                   VMS).  This can be used in place of -D.
+                                   VMS).  This can be used instead of using
+                                   -D (or what corresponds to that on your
+                                   compiler) in CPPFLAGS.
```

Comment:
This is not actually entirely accurate :-) Yes, it *can* be used instead of (or rather in addition to) using -D in CPPFLAGS, but not universally. As it stands now, on VMS it *has to* be used instead of corresponding flag in CPPFLAGS. I mean VMS user *has to* remove /DEFINE=(I,ME,MINE) from CPPFLAGS and move the list to CPPDEFINES. In other words using equivalent of -D in CPPFLAGS is [currently] not an option on VMS, and therefore CPPDEFINES can't be described with "can" there :-)